### ISSUE TITLE
Support for Bessel functions (or SpecialFunctions.jl in general)

### ISSUE THREAD

**Issue by merlinubc** [JuliaIntervals/IntervalRootFinding.jl#175]
It seems there is no ::Interval support for besselk and besseli of SpecialFunctions.jl since IntervalRootFinding.roots() gives a StackOverflowError when applied to (a combination of) these functions. This issue may be relevant for both IntervalArithmetic and IntervalRootFinding.

**Comment by dpsanders** [comment]
Yes, unfortunately implementing interval versions of special functions is Non-trivial. 

However, many such functions are implemented in arb, so I think we should just wrap those.

**Comment by merlinubc** [comment]
That would be highly appreciated - since the performance of IntervalRootFinding is so strong for "normal" functions and Bessel functions appear often in boundary value problems.

**Comment by wsshin** [comment]
I am suffering from this exact issue as well.  It will be very nice to be able to find the roots of special functions by `IntervalRootFinding.jl`.
